main.gms

```diff
@@ -1407,6 +1407,16 @@ $setGlobal cm_import_ariadne  off !! def off
 *** then the values from the region group disaggregation will be overwritten by this region-specific value.
 *** For example: "2030.2050.MEA.EU27_regi.seh2 0.5, 2030.2050.MEA.DEU.seh2 0.3".
 $setGlobal cm_trade_SE_exog off !! def off
+*** This allows to manually adjust the ramp-up curve of the SE tax on electricity. It is mainly used for taxing electricity going into electrolysis for green hydrogen production.
+*** The ramp-up curve is a logistic function that determines how fast taxes increase with increasing share of technology in total power demand.
+*** This essentially makes an assumption about to what extend the power demand of electrolysis will be taxed and how much tax exemptions there will be at low shares of green hydrogen production.
+*** The parameter a defines how fast the tax increases with increasing share, while the parameter b defines at which share
```

Comment:
I would make it more precise: "The parameter a defines how fast the tax increases with increasing share, with a*100 being the percentage point range over which the tax value increases from 12% to 88%"

Reply:
https://github.com/remindmodel/remind/pull/1668/commits/1b86906386c55ecbf44005eb1b1f2266379f771d